packages/web3torrent/src/utils/calculateWei.ts

```diff
@@ -1,12 +1,18 @@
 import {WEI_PER_BYTE} from '../library/web3torrent-lib';
 import {utils} from 'ethers';
+import prettier from 'prettier-bytes';
 
 export const calculateWei = (fileSize: number | string) => {
   if (!isNaN(Number(fileSize))) {
     return WEI_PER_BYTE.mul(fileSize);
   } else return utils.bigNumberify(NaN);
 };
 
+export const prettyPrintBytes = (wei: utils.BigNumber): string => {
+  const bytes = wei.div(WEI_PER_BYTE);
+  return prettier(bytes.toNumber());
+};
+
```

Comment:
Seems nice to me to have a function `calculateBytes` or `calculateBytesFromWei` that just does the unit conversion, and no formatting. 

The `prettier()` call could then be (optionally) called in the context where `calculateBytes` is called.

This would match `prettyPrintWei(calculateWei(...))`

Reply:
Yeah. I thought about that, but we had another prettier function down below, so I follow that. The case where we want `calculate` not `prettier` is not clear yet, so I didn't bother to refactor it.